### PR TITLE
dbus: update to 1.13.18

### PR DESCRIPTION
**Pull request by Yisus7u7**

### CODE REVIEW

**Comment by Yisus7u7** [comment]
@xeffyr 
After this update, many packages in x11-packages will have to be rebuilt, since most use the services of dbus.

**Comment by Grimler91** [comment]
Please try to use more informative commit messages:

`Update some packages` does not say what packages are involved, and gives the impression that multiple packages are being updated (rather than just dbus). `dbus: update to 1.13.18` is a better description of what the commit does.

`Fix script` gives the impression that you are fixing a script in the scripts/ directory rather than the build.sh recipe for dbus. This commit should/could in any case be squashed with the first one.

File: packages/dbus/build.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@ TERMUX_PKG_HOMEPAGE=https://dbus.freedesktop.org
 TERMUX_PKG_DESCRIPTION="Freedesktop.org message bus system"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="Leonid Pliushch <leonid.pliushch@gmail.com>"
-TERMUX_PKG_VERSION=1.12.20
-TERMUX_PKG_REVISION=3
-TERMUX_PKG_SRCURL="https://dbus.freedesktop.org/releases/dbus/dbus-$TERMUX_PKG_VERSION.tar.gz"
-TERMUX_PKG_SHA256=f77620140ecb4cdc67f37fb444f8a6bea70b5b6461f12f1cbe2cec60fa7de5fe
+TERMUX_PKG_VERSION=1.13.18
+TERMUX_PKG_REVISION=4
```
Comment (Grimler91) [inline packages/dbus/build.sh:6]: No need to bump revision. The line can be removed when increasing TERMUX_PKG_VERSION

**Review by Grimler91 — CHANGES_REQUESTED** [review]

**Comment by Yisus7u7** [comment]
Done

**Comment by Grimler91** [comment]
The TERMUX_PKG_REVISION line can be removed, and I would prefer if you squash the commits into one, see for example https://stackoverflow.com/a/50443893 on how to do that

**Comment by Yisus7u7** [comment]
> The TERMUX_PKG_REVISION line can be removed, and I would prefer if you squash the commits into one, see for example https://stackoverflow.com/a/50443893 on how to do that

xeffyr had explained to me that every time a package is modified, TERMUX_PKG_REVISION had to be increased, no matter what modification it was.

**Comment by Yisus7u7** [comment]
> The TERMUX_PKG_REVISION line can be removed, and I would prefer if you squash the commits into one, see for example https://stackoverflow.com/a/50443893 on how to do that

This is not the "master" branch, therefore if the change works it will only be a confirmation and not all the ones I did.

**Comment by Grimler91** [comment]
> > The TERMUX_PKG_REVISION line can be removed, and I would prefer if you squash the commits into one, see for example https://stackoverflow.com/a/50443893 on how to do that
> 
> xeffyr had explained to me that every time a package is modified, TERMUX_PKG_REVISION had to be increased, no matter what modification it was.

Only if TERMUX_PKG_VERSION is not changed

> > The TERMUX_PKG_REVISION line can be removed, and I would prefer if you squash the commits into one, see for example https://stackoverflow.com/a/50443893 on how to do that
> 
> This is not the "master" branch, therefore if the change is merged it will only be a confirmation and not all the ones I did.

I don't understand what you are trying to say here unfortunately, what "confirmation"? It doesn't matter that your branch is not master, you can do 
```
git clone https://github.com/Yisus7u7/termux-packages -b patch-1
### Or, if you have uploaded an ssh pubkey to github:
# git clone git@github.com:Yisus7u7/termux-packages -b patch-1
cd termux-packages
git rebase -i HEAD~3
```
And follow instructions in linked post to squash the three commits into one. You can then force push to your branch to update this PR:
```
git push -f https://github.com/Yisus7u7/termux-packages patch-1
### Or:
# git push -f git@github.com:Yisus7u7/termux-packages patch-1

**Review by xeffyr — APPROVED** [review]

**Comment by Yisus7u7** [comment]
@xeffyr
You can update the submodule in x11-packages?, do you have to update now many packages that depend on dbus

**Pull request merged**